src/odata-to-abstract-sql.ts

```diff
@@ -1310,7 +1312,7 @@ export class OData2AbstractSQL {
 		}
 	}
 	DurationMatch(match: DurationNode[1]): AbstractSqlType | undefined {
-		if (!_.isObject(match)) {
+		if (match == null || typeof match !== 'object') {
```

Comment:
Should we instead trust the typings that say this is always an object and just drop this `if` completely?

Reply:
I wouldn't trust the typings as being followed; they're what's intended to be passed in but I would not be surprised at all if it's not. After all this module and the one that feeds into it was in OMeta initially and the one that feeds into it is now in pegjs so it's still not particularly typed